cloudslang-runtime/src/main/java/io/cloudslang/lang/runtime/steps/ActionExecutionData.java

```diff
@@ -245,8 +292,14 @@ private String normalizeJavaGav(String gav) {
     private Map<String, Value> prepareAndRunPythonAction(Collection<String> dependencies, String pythonScript,
                                                          Map<String, Value> callArguments, boolean useJython) {
         if (StringUtils.isNotBlank(pythonScript)) {
-            return scriptExecutor.executeScript(
+            final Map<String, Value> returnedMap = scriptExecutor.executeScript(
                     normalizePythonDependencies(dependencies), pythonScript, callArguments, useJython);
+
+            if (returnedMap.containsKey(EXCEPTION)) {
```

Comment:
isNotEmpty check seems better as you can have a null or "" value for a key.

Reply:
the value is not a String as the isNotEmpty function expects